public/app/features/datasources/components/DataSourceTestingStatus.tsx

```diff
@@ -100,7 +100,7 @@ export function DataSourceTestingStatus({ testingStatus, exploreUrl, dataSource
   if (message) {
     return (
       <div className="gf-form-group p-t-2">
-        <Alert severity={severity} title={message} aria-label={e2eSelectors.pages.DataSource.alert}>
+        <Alert severity={severity} title={message}>
```

Comment:
Did we not need the data-testid attribute for this?

Reply:
I am not sure. I thought we don't but some failing e2e might contradict.

Furthermore, the <Alert> component itself is already adding its own data-testid so adding it here will overwrite it.